L1Trigger/TrackFindingTracklet/data/processingmodules_hourglassExtendedAllCombined.dat

```diff
@@ -0,0 +1,241 @@
+FitTrack:                    FT_D1D2
```

Comment:
Why do all extended txt files exist both in a "Combined" and an "AllCombined" version?

Reply:
When I updated the extended wiring to include only the combined modules, I added the new wiring with the "AllCombined" suffix. I did it this way so as not to break anything in case the cms-data repo was updated before the cms-sw repo, since the "AllCombined" wiring required a few modifications in the C++.

However, the old "Combined" wiring shouldn't be used anymore, and I have removed those files here.